mcs_benchmark_data/transformers/commonsenseQA_transformer.py

```diff
@@ -1,231 +1,140 @@
 from datetime import datetime
-from typing import Dict, Optional
+from typing import Dict, Optional, Tuple, Generator
+from pathlib import Path
 from urllib.parse import quote_plus
 import json 
+import os
 
 from rdflib import Graph, Literal, URIRef
 from rdflib.namespace import RDF
 
 from mcs_benchmark_data._transformer import _Transformer
-from mcs_benchmark_data.models.institution import Institution
-from mcs_benchmark_data.models.object import Object
-from mcs_benchmark_data.models.person import Person
+from mcs_benchmark_data._model import _Model
 from mcs_benchmark_data.namespace import MCS, SCHEMA
+from mcs_benchmark_data.models.benchmark import Benchmark
+from mcs_benchmark_data.models.benchmark_bootstrap import BenchmarkBootstrap
+from mcs_benchmark_data.models.benchmark_dataset import BenchmarkDataset
+from mcs_benchmark_data.models.benchmark_question import BenchmarkQuestion
+from mcs_benchmark_data.models.benchmark_choice import BenchmarkChoice
+from mcs_benchmark_data.models.benchmark_choices import BenchmarkChoices
+from mcs_benchmark_data.models.submission import Submission
+from mcs_benchmark_data.models.submission_sample import SubmissionSample
+from mcs_benchmark_data.models.benchmark_question_type import BenchmarkQuestionType
+
+class CommonsenseQaTransformer(_Transformer):
+    __TYPE = "@type"
+    __LIST = "itemListElement"
+
+    def transform(self, *, benchmark_json_file_path: Path, dev_json_file_path: Path,
+     train_json_file_path: Path, submission_json_file_paths: Tuple[Path]
+    ) -> Generator[_Model, None, None]:
+        benchmark_json = open(benchmark_json_file_path)
+        self.__transform_benchmark(benchmark_json_file_path)
+
+        #Yield benchmark dev sample
+        dev_json = open(dev_json_file_path)
+        self.__transform_benchmark_sample(dev_json, "dev")
+
+        #Yield benchmark train sample
+        train_json = open(train_json_file_path)
+        self.__transform_benchmark_sample(train_json, "train")
+
+        #Yield submissions
+        #Assumes file name in form "*_[systemname]_submission.jsonl" (e.g. dev_rand_split_roberta_submission.jsonl)
+        for path in submission_json_file_paths:
+            submission_json = open(path)
+            dirs, fname = os.path.split(path)
+            system = fname.split("_")[-2]
+            self.__transform_submission(submission_json, system)
+
+    def __transform_benchmark(self, benchmark_json):
+        #benchmark_bootstrap = BenchmarkBoostrap.from_json(benchmark_json)
+        benchmark_metadata = json.loads(benchmark_json)

```

Comment:
If you pass in benchmark_json_file_path here you can do

```
with open(benchmark_json_file_path) as benchmark_json_file:
    benchmark_metadata = json.load(benchmark_json_file)
```

Notice .load and not .loads.